scene/resources/syntax_highlighter.h

```diff
@@ -56,6 +63,8 @@ class SyntaxHighlighter : public Resource {
 	Dictionary get_line_syntax_highlighting(int p_line);
 	virtual Dictionary _get_line_syntax_highlighting_impl(int p_line) { return Dictionary(); }
 
+	void clear_line_highlighting_cache(int p_line);
```

Comment:
could this be useful to expose? How is changing 1 line handled that could span multiple lines of styling?

For example `"` can span multiple lines, would we need to clear each line after that character for the styling to apply?